eng/native/naming.props

```diff
@@ -45,10 +36,4 @@
     <AdditionalSymbolPackageExcludes Condition="'$(LibSuffix)' != ''" Include="%2A%2A\%2A.a;%2A%2A\%2A$(LibSuffix)" />
   </ItemGroup>
 
-  <!-- arcade is using long name for this property; 'SymbolFileExtension'.
```

Comment:
I couldnt find reference to either in arcade.

Reply:
Yup, it was removed from arcade in https://github.com/dotnet/arcade/commit/99968bd08a1d0a031a557d40557028b05ce8c2fc and https://github.com/dotnet/runtime/pull/73269 cleaned it up in the runtime.